Examples/Swift/OfflinePackExample.swift

```diff
@@ -101,6 +100,12 @@ class OfflinePackExample_Swift: UIViewController, MGLMapViewDelegate {
             if completedResources == expectedResources {
                 let byteCount = ByteCountFormatter.string(fromByteCount: Int64(pack.progress.countOfBytesCompleted), countStyle: ByteCountFormatter.CountStyle.memory)
                 print("Offline pack “\(userInfo["name"] ?? "unknown")” completed: \(byteCount), \(completedResources) resources")
+
+                // Remove notification handlers
+                NotificationCenter.default.removeObserver(self, name: NSNotification.Name.MGLOfflinePackProgressChanged, object: nil)
+                NotificationCenter.default.removeObserver(self, name: NSNotification.Name.MGLOfflinePackError, object: nil)
+                NotificationCenter.default.removeObserver(self, name: NSNotification.Name.MGLOfflinePackMaximumMapboxTilesReached, object: nil)
```

Comment:
Each of these observations would be worth removing during deinitialization too. Similar to what you did with the calls to `addObserver(_:selector:name:object:)`, you can factor these calls to `removeObserver(_:name:object:)` into a separate method for consistency.

Reply:
oddly enough, changing `completedResources == expectedResources` to `pack.state == .complete` resolves the duplicate notification in this example